src/Tribe/API.php

```diff
@@ -270,6 +270,40 @@ public static function update_event_cost( $event_id ) {
 		}
 
 		/**
+		 * Update user-created Additional Fields
+		 *
+		 * @param $event_id
+		 * @param $data
+		 */
+		public static function update_additional_fields ( $event_id, $data ) {
+			$additionalFieldTags = array_map(
+				function( $field ) { return $field['name']; },
```

Comment:
I would love for you to use a structure for variables like the following:
E.g.: `$additional_field_tags`